spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -201,87 +204,53 @@ function onChanOpenAck(
   // port has already been validated
   // assert that version is "ics27-1"
   abortTransactionUnless(version === "ics27-1")
+  // state change to keep track of successfully registered interchain account
+  setActiveChannel(SourcePortId)
 }
 ```
 
 ```typescript
 function onChanOpenConfirm(
   portIdentifier: Identifier,
   channelIdentifier: Identifier) {
-  // accept channel confirmations, port has already been validated
+  setActiveChannel(CounterPartyPortId)
 }
 ```
 
 ```typescript
 function onChanCloseInit(
   portIdentifier: Identifier,
   channelIdentifier: Identifier) {
-  // no action necessary
+  unsetActiveChannel(CounterPartyPortId)
```

Comment:
delete interchain account as well

Reply:
I think the idea is to leave the interchain account and allow it to be associated with a different active channel